spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/lock/JdbcLockRegistryTests-context.xml

```diff
@@ -24,8 +24,6 @@
 
 	<bean id="lockClient" class="org.springframework.integration.jdbc.lock.DefaultLockRepository">
 		<constructor-arg name="dataSource" ref="dataSource"/>
-		<property name="insertQuery"
-				  value="INSERT INTO INT_LOCK (REGION, LOCK_KEY, CLIENT_ID, CREATED_DATE) VALUES (?, ?, ?, ?)"/>

```

Comment:
There was a reason to have this in the test: https://github.com/spring-projects/spring-integration/issues/3866.

Even if it is the same value we have by default, it still covers a respective setter call.
So, please, consider to revert your change.
It is still not relevant to the purpose of this PR.